src/VisualStudio/Core/Def/Implementation/DebuggerIntelliSense/DebuggerIntellisenseFilter.cs

```diff
@@ -44,6 +45,14 @@ internal void SetNextFilter(IOleCommandTarget nextFilter)
         internal void SetCommandHandlers(ITextBuffer buffer)
         {
             this.CurrentHandlers = _commandFactory.GetService(buffer);
+            // Chain in editor command handler service. It will execute all our command handlers migrated to the modern editor commanding
+            // on the same text view and buffer as this.CurrentHandlers.
+            var componentModel = (IComponentModel)_languageService.SystemServiceProvider.GetService(typeof(SComponentModel));
+            var vsCommandHandlerServiceAdapterFactory = componentModel.GetService<IVsCommandHandlerServiceAdapterFactory>();
+            var vsCommandHandlerServiceAdapter = vsCommandHandlerServiceAdapterFactory.Create(ConvertTextView(), 
+                GetSubjectBufferContainingCaret(), // our override doesn't actually check the caret and always returns _context.Buffer
+                NextCommandTarget);
+            NextCommandTarget = vsCommandHandlerServiceAdapter;
```

Comment:
Good catch. There is a weird dependency between ```SetNextFilter()``` and ```SetContext()/SetCommandHandlers()``` here, but I found a way to unify them so that NextCommadTarget is set in only one place